### ISSUE TITLE
Document the unicode autocompletion feature

### ISSUE THREAD

**Issue by joshop** [sharkdp/numbat#586]
I never knew about the `\alpha` + TAB to get `α`, etc. feature in the CLI before reading an issue that mentioned it, shouldn't this be documented somewhere? It seems nice to have but I can see most people not even discovering that it exists.
Also, this is probably outside the scope of the same issue but are there plans to add a similar feature to the web version?

**Comment by sharkdp** [comment]
> I never knew about the \alpha + TAB to get α, etc. feature in the CLI before reading an issue that mentioned it, shouldn't this be documented somewhere?

Yes, that is a good idea — thank you. The Unicode-input feature is inspired by [Julia](https://docs.julialang.org/en/v1/manual/unicode-input/
). It's probably a good idea to either link to the [implementation](https://github.com/sharkdp/numbat/blob/b2410caac914839ee4d0e4fc3153a7cb2fe5a742/numbat/src/unicode_input.rs) as a reference, or to use that `UNICODE_INPUT` array to auto-generate the reference documentation.

> are there plans to add a similar feature to the web version?

I don't really remember if I tried it shortly and failed (I think so), or if I simply forgot to implement it in the web version. In any case: yes, it would be great to have this in the web version as well.

**Comment by sharkdp** [comment]
This feature is now available in the web version.

It's also documented here: https://numbat.dev/docs/unicode-input/

closed via https://github.com/sharkdp/numbat/pull/787
